README.md

```diff
@@ -22,29 +22,8 @@ Helps discover good candidates for refactoring.
 * Creates a score based on these two values.
 
 The results are displayed in a table:
-```
-    ___  _   _  __  __  ____  _  _     ____  _   _  ____
-   / __)( )_( )(  )(  )(  _ \( \( )___(  _ \( )_( )(  _ \
-  ( (__  ) _ (  )(__)(  )   / )  ((___))___/ ) _ (  )___/
-   \___)(_) (_)(______)(_)\_)(_)\_)   (__)  (_) (_)(__)      https://github.com/bmitch/churn-php
-
-+---------------------------------------------------------------------+---------------+------------+-------+
-| File                                                                | Times Changed | Complexity | Score |
-+---------------------------------------------------------------------+---------------+------------+-------+
-| src/Managers/FileManager.php                                        | 5             | 4          | 9     |
-| src/Assessors/CyclomaticComplexity/CyclomaticComplexityAssessor.php | 4             | 4          | 8     |
-| src/Results/ResultsParser.php                                       | 3             | 3          | 6     |
-| src/Results/Result.php                                              | 2             | 1          | 3     |
-| src/Factories/ProcessFactory.php                                    | 2             | 1          | 3     |
-| src/Results/ResultCollection.php                                    | 1             | 1          | 2     |
-| src/Values/File.php                                                 | 1             | 1          | 2     |
-| src/Collections/FileCollection.php                                  | 1             | 1          | 2     |
-| src/Values/Config.php                                               | 1             | 1          | 2     |
-| src/Processes/ChurnProcess.php                                      | 1             | 1          | 2     |
-+---------------------------------------------------------------------+---------------+------------+-------+
-  10 files analysed in 0.24276995658875 seconds using 15 parallel jobs.
-```
 
+![](img/result.png)
```

Comment:
Thanks for the new screenshot but you are right, running it on a fresh Laravel install doesn't really show much variety. Would you be able to make the screenshot again but just run it on this package itself? So you'd get a screenshot similar to what I have currently?

Reply:
Sure